crates/wasmtime/src/store.rs

```diff
@@ -943,7 +941,7 @@ impl<T> Store<T> {
     /// for an introduction to epoch-based interruption.
     pub fn epoch_deadline_callback(
```

Comment:
Can the documentation for this method be updated to talk about how returning `Yield` will panic if `async_support` is disabled?